aws-android-sdk-core/src/main/java/com/amazonaws/auth/CognitoCachingCredentialsProvider.java

```diff
@@ -95,7 +95,7 @@
 
     private static final String TAG = "CognitoCachingCredentialsProvider";
 
-    private AWSKeyValueStore awsKeyValueStore;
+    AWSKeyValueStore awsKeyValueStore;
```

Comment:
Why the change?

Reply:
I am using this for testing. So made it package private.